docs/concepts/storage/storage-classes.md

```diff
@@ -52,6 +52,20 @@ mountOptions:
   - debug
 ```
 
+### Volume Binding Mode
+{% assign for_k8s_version="v1.9" %}{% include feature-state-alpha.md %}
+
+This field requires the `VolumeScheduling` feature gate to be enabled.
+
+A new field, `volumeBindingMode`, controls if volume binding should happen
+immediately or wait until pod scheduling.  Valid values are `Immediate` and
+`WaitForFirstConsumer`.  The default mode is `Immediate`.
```

Comment:
Consider clarifying that `Immediate` is equivalent to the behavior prior to 1.9.

Reply:
done